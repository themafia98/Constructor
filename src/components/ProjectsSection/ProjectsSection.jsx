import React from 'react';

import List from '../List/List.jsx';

import './ProjectsSection.scss';

const ProjectsSection = props => {

        return (
                <section className = 'projectsList'>
                    <div className = 'container'>
                        <div className = 'flex-column'>
                            <div className = 'projectsList__about'>
                                <h3>Your Projects</h3>
                            </div>
                            {List ? <List /> : null}
                        </div>
                    </div>
                </section>
        )

}

export default ProjectsSection;