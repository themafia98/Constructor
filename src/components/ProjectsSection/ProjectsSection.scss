

.projectsList__about {
    h3 {
        text-align: center;
    }
}
