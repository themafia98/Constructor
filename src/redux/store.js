import {createStore} from 'redux';
import combineReducers from './reducers';

let broswerPluginDev = window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__();
const store = createStore(combineReducers,broswerPluginDev);
export default store;