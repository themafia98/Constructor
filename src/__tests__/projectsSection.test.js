import React from 'react';
import {Provider} from 'react-redux';
import ProjectsSection from '../components/ProjectsSection/ProjectsSection';

import renderer from 'react-test-renderer';

import store from '../redux/testReduxStore';

const ProjectSestion = renderer.create(
    <Provider store = {store}>
        <ProjectsSection />
    </Provider>
)

test('ProjectsSection.jsx test ', function(){

    let snapshotProjectsSestion = ProjectSestion.toJSON();
    expect(snapshotProjectsSestion).toMatchSnapshot();
});