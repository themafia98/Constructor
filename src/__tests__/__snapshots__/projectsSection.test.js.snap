// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectsSection.jsx test  1`] = `
<section
  className="projectsList"
>
  <div
    className="container"
  >
    <div
      className="flex-column"
    >
      <div
        className="projectsList__about"
      >
        <h3>
          Your Projects
        </h3>
      </div>
      <div
        className="projectsList__list"
      />
    </div>
  </div>
</section>
`;
